controllers/articles.controller.js

```diff
@@ -0,0 +1,10 @@
+const { fetchArticlesById } = require('../models/articles.model');
+
+exports.getArticlesById = (req, res, next) => {
+  const id = req.params.article_id;
+  fetchArticlesById(id)
+    .then((article) => {
+      res.status(200).send(article);
```

Comment:
Can you please send the response on an object with a key representing what it holds

E.g. `res.status(200).send({article})`

This is common practice and will make your life easier when doing the front end project!